controllers/upgrade.js

```diff
@@ -0,0 +1,45 @@
+const PouchDB = require('pouchdb-core');
+PouchDB.plugin(require('pouchdb-adapter-http'));
+
+const buildDbUrl = 'https://staging.dev.medicmobile.org/_couch/builds';
+const buildDb = new PouchDB(buildDbUrl);
+const targetDbUrl = process.env.COUCH_URL;
+const targetDb = new PouchDB(targetDbUrl);
+
+module.exports = (version, username) => {
+  console.log('upgrade()', `Upgrading to ${version}…`);
+
+  console.log('upgrade()', 'Fetching newDdoc…');
+  return buildDb
+    .get(version, { attachments:true })
+    .then(newDdoc => console.log('upgrade()', 'Fetched newDdoc.') || newDdoc)
+
+    .then(newDdoc => console.log('upgrade()', 'Fetching oldDdoc…') || newDdoc)
+    .then(newDdoc =>
+      targetDb.get('_design/medic')
```

Comment:
Again I think we'll want to be able to upgrade other apps here so expose the `medic` bit in the API as the app name. Alternatively use the _id from `newDdoc`.